.github/workflows/linux-eic-shell.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - 'v1.*' # maintenance branches
```

Comment:
```suggestion
      - 'v*.*' # maintenance branches
```
?

Reply:
Apparently, it even supports some regex-like https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
```suggestion
      - 'v[0-9]+.[0-9]+' # maintenance branches
```

Reply:
Yeah, I wasn't sure. I thought it just did limited globbing.